daml-lf/spec/daml-lf-1.rst

```diff
@@ -584,8 +584,8 @@ Then we can define our kinds, types, and expressions::
        |  e.f                                       -- ExpTupleProj: Tuple projection
        |  ⟨ e₁ 'with' f = e₂ ⟩                      -- ExpTupleUpdate: Tuple update
        |  u                                         -- ExpUpdate: Update expression
-       | 'to_any_template' @Mod:T t                 -- ExpToAnyTemplate: Wrap a template in AnyTemplate
-       | 'from_any_template' @Mod:T t               -- ExpToAnyTemplate: Extract the given template from AnyTemplate or return None
+       | 'to_any' @τ t                              -- ExpToAny: Wrap a value of the given type in Any
```

Comment:
Naive question. 
 Why not `'to_any' @τ : τ -> Any` instead of `'to_any' @τ e: Any`  ?

Reply:
Two reasons:
1. For `to_any`, you would end up with applications sometimes being values if the function is `to_any τ` which is a bit awkward. This does not apply to `from_any`.
2. Consistency: Changing only `from_any` while keeping `to_any` the way it is due to 1 feels weird and also doesn’t fit in with things like `Cons`, `create`, … that are all defined similarly.